frontend/src/_app-theme.scss

```diff
@@ -53,4 +53,33 @@
   .hoverable:hover {
     background-color: mat-color($foreground, base, 0.04);
   }
+
```

Comment:
The rest of the changes look good, but I'm hesitant to accept these global style changes - what exactly are these supposed to affect?

Reply:
Sorry, forgot to move the table ones to the appropriate files.

The `body.avr-app-background` in the global is to make it so that the entire website isn't stretched out on wide monitors.